composer.lock

```diff
@@ -0,0 +1,1425 @@
+{
```

Comment:
The benefit of committing the lock file is not so much for e.g. CI, since I do `update` there anyway (since this is a lib that will be installed in a variety of different dep configurations); but it is that anyone else can know exactly what works for the maintainers of the package (including sharing this between maintainers themselves), even if some other configurations for some reason don't.

But again, if you think this is not a good idea, I will remove.